packages/common/src/ui/layout/tables/RemoteDataTable.tsx

```diff
@@ -76,39 +70,33 @@ export const RemoteDataTable = <T extends Record<string, unknown>>({
     >
       <MuiTable sx={{ display: 'block', overflow: 'auto' }}>
         <TableHead sx={{ flex: 1, display: 'flex', flexDirection: 'column' }}>
-          {headerGroups.map(({ getHeaderGroupProps, headers }) => (
-            <HeaderRow key={getHeaderGroupProps().key}>
-              {headers.map(column => (
-                <HeaderCell
-                  minWidth={column.minWidth}
-                  width={Number(column.width)}
-                  onSortBy={onSortBy}
-                  key={column.getHeaderProps().key}
-                  isSortable={!column.disableSortBy}
-                  isSorted={column.id === sortBy.key}
-                  align={column.align}
-                  id={column.id as KeyOf<T>}
-                  direction={sortBy.direction}
-                >
-                  {column.render('Header')}
-                </HeaderCell>
-              ))}
-            </HeaderRow>
-          ))}
+          <HeaderRow>
+            {columns.map(column => (
+              <HeaderCell
+                minWidth={column.minWidth}
+                width={Number(column.width)}
+                onSortBy={onSortBy}
+                key={String(column.key)}
+                isSortable={!!column.sortable}
+                isSorted={column.key === sortBy.key}
+                align={column.align}
+                columnKey={column.key}
+                direction={sortBy.direction}
+              >
+                <column.Header column={column} />
+              </HeaderCell>
+            ))}
+          </HeaderRow>
```

Comment:
I took away the headergroup and now just rendering `column.Header` directly.

Looking at this now, maybe would be better to just pass the column object straight to the `Cell`, that's likely easier

Reply:
Will do this also